Podcast/ListeningHistoryViewController.swift

```diff
@@ -115,20 +112,33 @@ class ListeningHistoryViewController: ViewController, UITableViewDelegate, UITab
     //MARK
     //MARK - Endpoint Requests
     //MARK
+    func emptyStateTableViewHandleRefresh() {
+        fetchEpisodes(refresh: true)
+    }
 
-    @objc func fetchEpisodes(refresh: Bool = true) {
+    func fetchEpisodes(refresh: Bool) {
```

Comment:
is `refresh` ever set to false? might be easier to have a default value of true if it's usually true